config/jobs/kubernetes-sigs/cluster-api/cluster-api-periodics-release-1-1.yaml

```diff
@@ -0,0 +1,213 @@
+periodics:
+- name: periodic-cluster-api-test-release-1-1
+  interval: 1h
```

Comment:
Can you please adjust the intervals to the ones we have in the corresponding jobs in release-1-0.yaml?
(the idea is that we run the jobs on release branches less frequent than on main)